Translations/translation_BG.json

```diff
@@ -288,7 +288,21 @@
 				"Minimum",
 				"voltage"
 			],
-			"desc": "Minimum allowed voltage per cell"
+			"desc": "Minimum allowed voltage per cell <Volts> <3S — 3.0V - 3.7V, 4/5/6S — 2.4V - 3.7V>"
```

Comment:
This change of string looks like a merge error, same for other languages.

Reply:
That's not error, I forgot to add this in previous PR

Reply:
Then I think these should be in a separate PR...

Reply:
You are right, but if Ralim does not accept this PR, then I will create it separately

Reply:
Please put language changes in their own PR